packages/dds/sequence/README.md

```diff
@@ -131,12 +168,23 @@ Both markers and text are stored as segments in the Shared String. Text segments
   - [collaborativeTextArea](https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/react-inputs/src/CollaborativeTextArea.tsx)
   - [collaborativeInput](https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/react-inputs/src/collaborativeInput.tsx)
 
-# Sparse Matrix
+## SparseMatrix (obsolete)
+
+**OBSOLETE:** SparseMatrix has been replaced by SharedMatrix.
```

Comment:
@anthony-murphy @DLehenbauer Is SparseMatrix obsolete?

Reply:
Correct.  I'd consider omitting SparseMatrix from the README.  It's replacement (SharedMatrix) is better in every way.

Reply:
I'll remove the docs from the readme and filed a follow up to mark the code itself as obsolete in the TSDocs.

Reply:
but it is present in code [packages/dds/sequence/src/sparsematrix.ts](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/sequence/src/sparsematrix.ts) so do need to remove it everywhere or only from docs ?